modules/doi/includes/utilities.inc

```diff
@@ -864,3 +864,32 @@ function islandora_doi_get_mods($id) {
 
   return islandora_doi_translate($source_string, $content_type);
 }
+
+/**
+ * Strip HTML tags from imported XML metadata.
```

Comment:
Should they not just be escaped?

Reply:
@willtp87 I misspoke in that function description. The function is in fact escaping any subelements, for example changing `<i>this text</i>` to `&lt;i&gt;thistext&lt;/i&gt;`.